### PR TITLE
RELATED: RAIL-990 Table totals in AD

### PR DESCRIPTION
**Pull request by Hurtak**
- added AD support for pivot table totals
- improved tests
- added `s-` classes to pivot table cells and menu
- mockjs update

### CODE REVIEW

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2780/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2158/ : FAILURE in 8m 45s

**Comment by gdgate** [comment]
Build in pipeline check-extended aborted.

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2782/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2159/ : SUCCESS in 4m 37s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1140/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2783/ : SUCCESS in 10s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/675/ : SUCCESS in 19m 42s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2790/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2166/ : SUCCESS in 10m 19s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

File: examples/test/Menu_test.js [hunk 1]
```diff
@@ -53,28 +53,32 @@ fixture('Menu')
...
 
+    await t.expect(menuContentItem.exists).eql(false);
     await t.hover(measureCell);
-    await t.expect(menu.visible).eql(true);
 
-    const anotherCell = getMeasureGroupCell(0);
-    await t.hover(anotherCell);
-    await t.expect(menu.visible).eql(false);
+    await t.click(menuToggler);
+    await t.expect(menuContentItem.exists).eql(true);
```
[the remark this answers is not in the document]
Comment (Hurtak) [reply examples/test/Menu_test.js]: Line removed

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2792/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2168/ : FAILURE in 1m 17s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2793/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2169/ : SUCCESS in 5m 14s

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2794/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2170/ : SUCCESS in 8m 04s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1143/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2795/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/678/ : UNSTABLE in 21m 07s

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1144/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build failed (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2797/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/679/ : UNSTABLE in 21m 20s

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2801/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2175/ : SUCCESS in 3m 53s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1145/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2802/ : SUCCESS in 11s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/680/ : SUCCESS in 25m 33s

**Comment by Hurtak** [comment]
extended test - storybook

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2807/ : SUCCESS in 17s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2179/ : SUCCESS in 13m 43s

**Comment by yenkins** [comment]
Link to Storybook test result:

https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1147/artifact/test-storybook/html-report/index.html

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2808/ : SUCCESS in 8s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-pull-request-dispatcher/682/ : SUCCESS in 27m 34s

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/602/ : SUCCESS in 36s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2187/ : FAILURE in 18m 05s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1150/ : ABORTED in 34m 32s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/574/ : SUCCESS in 23m 00s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-opensource-repository-check/2818/ : SUCCESS in 12s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2195/ : SUCCESS in 18m 02s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-react-components-merge-compliance-check/609/ : SUCCESS in 26s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-unit-tests-zuul-docker-el7/2205/ : SUCCESS in 2m 22s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-storybook-zuul-docker/1161/ : SUCCESS in 18m 45s
- https://checklist.intgdc.com/job/client-libs/job/gooddata-react-components-examples-zuul-docker/581/ : SUCCESS in 18m 48s

**Pull request merged**